src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -0,0 +1,51 @@
+namespace System
+{
+    internal static class HexConverter
+    {
+        public static char ToCharUpper(int value)
+        {
+            value &= 0xf;
+            return (char)(value > 9 ? value - 10 + 'A' : value + '0');
+        }
+
+        public static char ToCharLower(int value)
+        {
+            value &= 0xf;
+            return (char)(value > 9 ? value - 10 + 'a' : value + '0');
+        }
+
+#if NETCOREAPP || NETSTANDARD2_1
```

Comment:
One thing I've found is that as we move on to newer/more complex builds we keep needing to add to these.  Prefer expressing it negatively (i.e. `#if !NETSTANDARD2_0 && !NET48 && ...`), one for each flavor that fails.

This is because we expect few new build configurations to not want this, but a newer netstandard build would want it.

Reply:
I'll try to go with `NETCOREAPP` only. I am not sure how many NET48 version I'd need to list as for example NET48 is not used anywhere.